src/components/Swap/Summary/Estimate.tsx

```diff
@@ -19,11 +20,19 @@ interface EstimateProps {
 }
 
 export default function SwapInputOutputEstimate({ trade, slippage }: EstimateProps) {
-  const { estimateMessage } = useMemo(() => getEstimateMessage(trade, slippage), [slippage, trade])
+  const { estimateMessage } = useMemo(() => getEstimateMessage(trade, slippage, undefined), [slippage, trade])
```

Comment:
nit: can we put an inline comment here in front of undefined like `/** priceImpact */`?

also might be worth renaming the var to priceImpact for clarity?